tests/vmi_configuration_test.go

```diff
@@ -949,6 +957,41 @@ var _ = Describe("Configurations", func() {
 				}, 10*time.Second)
 			})
 		})
+
+		Context("when CPU features defined", func() {
+			It("should start a Virtaul Machine with matching features", func() {
+				vmiModel := "Conroe"
```

Comment:
is this test going to break if someone attempts to run this test on something that isn't Conroe or Opteron_G1?

Reply:
actually the cpu model here can be removed completely.